### DISCUSSION TITLE
永久外链需要改进

### DISCUSSION THREAD

**Discussion by iskycc** [spencerwooo/onedrive-vercel-index#489] [Ideas]
永久外链无法直接用Internet Download Manager下载
![image](https://user-images.githubusercontent.com/81439128/155095245-3e3bac9b-ab97-48ad-9c3d-bf913797135a.png)

使用linux的wget直接下载时(wget url)文件名和实际文件名不一致
![image](https://user-images.githubusercontent.com/81439128/155095595-3ec33463-dce5-44c7-9d9f-e427675efb4f.png)

**Comment by spencerwooo** [comment]
cc @myl7，应该也是为了把链接转成更可读的 URL 时候导致的。相关讨论：#483、#481
